pkgs/tools/misc/flameshot/default.nix

```diff
@@ -1,11 +1,11 @@
-{ stdenv, fetchFromGitHub, qtbase, qmake, qttools, qtsvg }:
+{ mkDerivation, lib, fetchFromGitHub, qtbase, qmake, qttools, qtsvg }:
 
 # To use `flameshot gui`, you will also need to put flameshot in `services.dbus.packages`
 # in configuration.nix so that the daemon gets launched properly:
 #
 #   services.dbus.packages = [ pkgs.flameshot ];
 #   environment.systemPackages = [ pkgs.flameshot ];
-stdenv.mkDerivation rec {
+mkDerivation rec {
   name = "flameshot-${version}";
```

Comment:
you could use pname as well

Reply:
How important is that change? I would like to avoid unrelated changes.

Reply:
don't mind it, you could have replaced `name = "flameshot-${version}";` with `pname = "flameshot"`. `name` is then generated by mkDerivation

Reply:
fixed.